spacy/language.py

```diff
@@ -1159,6 +1160,25 @@ def restore(self):
         self[:] = []
 
 
+def _count_pipeline_inter_dependencies(pipeline):
```

Comment:
Maybe this function should live in `analysis.py`? I have a bunch of similar-ish logic in there already.

Reply:
Yeah okay, I'll move it.